plugins/talk-plugin-featured-comments/client/containers/TabPane.js

```diff
@@ -0,0 +1,48 @@
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {compose, gql} from 'react-apollo';
+import TabPane from '../components/TabPane';
+import {withFragments} from 'plugin-api/beta/client/hocs';
+import {setActiveTab} from 'plugin-api/beta/client/actions/stream';
+
+const mapDispatchToProps = (dispatch) =>
+  bindActionCreators({
+    setActiveTab,
+  }, dispatch);
+
+const enhance = compose(
+  connect(null, mapDispatchToProps),
+  withFragments({
+    asset: gql`
+      fragment TalkFeatured_TabPane_asset on Asset {
+        id
+        featuredComments: comments(tags: ["FEATURED"], deep: true) {
+              nodes {
```

Comment:
The indentation varies throughout this fragment document, I think we should stick with 2 spaces for one indentation level